handlers/photo-haptics-audio-handler/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "photo-haptics-audio-handler",
+  "version": "1.0.0",
```

Comment:
Not really a problem since we don't initially use semver, but keep in mind that a 1.0.0 release implies it is stable!